App/Components/KeyboardResponsiveContainer.tsx

```diff
@@ -4,11 +4,13 @@ import {
   EmitterSubscription,
   View,
   ViewStyle,
-  LayoutAnimation
+  LayoutAnimation,
+  Platform
 } from 'react-native'
 
 interface Props {
   style?: ViewStyle
+  ios?: boolean
```

Comment:
this view is used in 2 places. one where i need it not to run when on iOS, this was the second simplest way to handle it. the most simple hit s TSC error, so wasn't that simple actually